join.md

```diff
@@ -6,18 +6,73 @@
 
 Dr. Brown can supervise research for you to get experience to decide if you want to go to graduate school, for project credit (CSC499,491), and occaionally with funding.  
 
-If you are interested, please read the website to learn about the work and  [submit this form just for current URI students](https://forms.gle/u8JCvdRJaYLbc4To7). The form notifies Dr. Brown by e-mail and allows you to book a meeting.
+If you are interested, please read the website to learn about the work and  [submit this form just for current URI students](https://forms.gle/u8JCvdRJaYLbc4To7). The form notifies Dr. Brown by e-mail and allows you to book a meeting as an interview.
 
 
 Typically, I prefer to work with students who have taken CSC310  (with any professor) or CSC392 (with me). 
+
+### Current Open Projects
+::::{card-carousel} 2
+
+:::{card} Robust ML evaluation Developer tools
+
+**building and maintaining Python packages that facilitate other research in the lab**
+
+Example tasks:
+- add unit tests
+- improve documentation
+- add new features to support libraries to make other research easier
+
+Build skills in software engineering, project management, and working with other developers. 
+
+Prereq: CSC392 or demonstrated proficiency in git, bash, and IDEs
+
+:::
+:::{card} When do fair ML algorithms work? 
+
+**Empirical evaluation of the contexts where fair ML algorithms succeed and fail through intentionally designed biased data**
+
+Example tasks: 
+- write code that impelements tyes of biased synthetic data based on mathematical models in the literature
+- evalute parameter ranges that create/do not create bias interactions
+- analyze data on existing biases to determine advice for data scientists
+- add more ML models by implementing recent advances in fair ML or developing novel interventions
+- generate new types of biased data based on your own understanding of how social systems create inequity
+
```

Comment:
This looks good to me, main objective of generating synthetic data is covered and all other pointers are great!